.github/workflows/import/poll-logs.js

```diff
@@ -0,0 +1,270 @@
+    return LOCAL_DEBUG_ENTRIES;
+  }
+
+  console.log(`Fetching logs for site: ${siteName} from ${baseUrl}...`);
+  const initialUrl = `${baseUrl}?from=${fromParam}`;
+  const entries = [];
+  let totalFetched = 0;
+
+  try {
+    let nextUrl = initialUrl;
+    let requestCount = 0;
+    const maxRequests = 1000; // Safety break
+
+    while (nextUrl && requestCount < maxRequests) {
+      requestCount++;
+      console.log(`Fetching page ${requestCount} for ${siteName}: ${nextUrl}`);
+      const request = await fetch(nextUrl, {
+        method: 'GET',
+        headers: {
+          Authorization: `token ${env.AEM_LIVE_ADMIN_TOKEN}`,
+        },
+      });
+
+      if (!request.ok) {
+        console.error(
+          `Error fetching logs for ${siteName}: ${request.status} ${request.statusText}`
+        );
+        const errorBody = await request.text();
+        console.error(`Response body: ${errorBody}`);
+        throw new Error(`Failed to fetch logs: ${request.status}`);
+      }
+
+      const json = await request.json();
+
+      if (json.entries && json.entries.length > 0) {
+        entries.push(...json.entries);
+        totalFetched += json.entries.length;
+        console.log(
+          `Fetched ${json.entries.length} entries for ${siteName}. Total: ${totalFetched}`
+        );
+      } else {
+        console.log(
+          `No new entries found on page ${requestCount} for ${siteName}.`
+        );
+      }
+
+      nextUrl = json.links?.next;
+      if (!nextUrl) {
+        console.log(`No more pages found for ${siteName}.`);
+        break;
+      }
+    }
+
+    if (requestCount >= maxRequests) {
+      console.warn(
+        `Warning: Reached maximum request limit (${maxRequests}) for ${siteName}. Log data might be incomplete.`
+      );
+    }
+
+    if (LOCAL_RUN) {
+      // Save entries for local debugging as a JS file with a default export
+      fs.writeFileSync(
+        '.github/workflows/import/LOCAL_DEBUG_ENTRIES.js',
+        `export default ${JSON.stringify(
+          entries.map((entry) => ({
+            path: entry.path,
+            route: entry.route,
+            paths: entry.paths,
+          })),
+          null,
+          2
+        )};\n`
+      );
+    }
+
+    if (!entries.length) {
+      await slackNotification(`No log entries found for from ${FROM_PARAM}`);
+      return;
+    }
+
+    return entries;
+  } catch (err) {
+    console.error(`Error fetching or writing logs for site ${siteName}:`, err);
+    throw err; // Re-throw error
+  }
+}
+
+async function getLivePaths(entries) {
+  const livePaths = Array.from(
+    new Set(
+      entries
+        .filter((entry) => entry.route === 'live')
+        .flatMap((log) => [
+          log.path,
+          ...(Array.isArray(log.paths) ? log.paths : []),
+        ])
+        .filter(Boolean)
+    )
+  );
+  if(LOCAL_RUN) console.log("Live paths found: ", livePaths.length);
+  if (!LOCAL_RUN)
```

Comment:
Maybe this could work as well
```suggestion
  if(LOCAL_RUN) console.log("Live paths found: ", livePaths.length);
  else
```